.travis.yml

```diff
@@ -20,4 +23,4 @@ notifications:
 
   email: false
 
-sudo: false
+sudo: false
```

Comment:
Add new line to your end of file to not break eslint rules

Reply:
thank you marius